src/Logs/Log.php

```diff
@@ -104,9 +104,10 @@ protected function parseText(array &$matches = []): void
 
     protected function fillMatches(array $matches = []): void
     {
-        $this->datetime = static::parseDatetime($matches[static::$regexDatetimeKey] ?? null)?->tz(
-            config('log-viewer.timezone', config('app.timezone'))
-        );
+        $datetime = static::parseDateTime($matches['datetime'] ?? null);
+        $timezone = config('log-viewer.timezone', config('app.timezone', 'UTC')) ?? 'UTC';
+        $this->datetime = $datetime?->setTimezone($timezone);
+
```

Comment:
this changes the logic. Should be parsing `$matches[static::$regexDatetimeKey] ?? null`

Reply:
nope, still not resolved.

You replaced
```php
static::parseDatetime($matches[static::$regexDatetimeKey] ?? null)
```
with
```php
static::parseDateTime($matches['datetime'] ?? null)
```
which changes the logic and the ability for other log parser classes to use a different datetime key.

Reply:
Oops! Forgot this one